src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/BackupResolver.kt

```diff
@@ -132,6 +144,45 @@ class BackupResolver(private val aem: AemExtension) {
             else -> listOf()
         }
 
+    fun create(instances: Collection<LocalInstance>) = namedFile.apply { create(this, instances) }
```

Comment:
moved code from task to backup resolver / so that code is now reusable in a custom task scripting